src/Paramore.Brighter.MsSql/MsSqlSqlAuthUnitOfWork.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Data;
+using System.Data.Common;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+
+namespace Paramore.Brighter.MsSql
+{
+    /// <summary>
+    /// A connection provider for Sqlite 
+    /// </summary>
+    public class MsSqlSqlAuthUnitOfWork : RelationalDbTransactionProvider
```

Comment:
Should this be Sql Auth Unit of work?

Reply:
I was trying to change it to MsSqlUnitOfWork as we only have the one